### PR TITLE
[ mergify ] require 2 days to pass after last update of PR to merge it

### PR DESCRIPTION
**Pull request by ulysses4ever**
Per discussion on IRC. Some members of the team expressed a concern that PRs sometimes get merged too fast. A chillout time after review was a proposed as a countermeasure. This PR implements 2 day chillout period after last update of any PR against `master` (so, backports are not included in this new policy).

---
Please include the following checklist in your PR:

* [x] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#conventions).
* [x] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#changelog).
* [x] The documentation has been updated, if necessary.

Please also shortly describe how you tested your change. Bonus points for added tests!

### CODE REVIEW

**Comment by Mikolaj** [comment]
I wonder if this change affects the very PR. Does mergify take its config from master or from current branch?

**Comment by ulysses4ever** [comment]
@Mikolaj I'd expect from master

**Comment by ulysses4ever** [comment]
Yes, unfortunately, I don't see any other way.

**Comment by jneira** [comment]
> Yes, unfortunately, I don't see any other way.

well making the experiment in a fork before merging upstream 😄
i usually pollute my fork master to do ci experiments

**Comment by Mikolaj** [comment]
@jneira: don't kill the suspense ;P

But isn't mergify installation somehow org-wide (Haskell org in this case)? Does it really work fine in a fork? I've never tried.

**Comment by ulysses4ever** [comment]
Can't self approve... I invited @jneira to help with that: https://github.com/ulysses4ever/cabal/pull/2

**Comment by ulysses4ever** [comment]
@jneira actully, this one: https://github.com/ulysses4ever/cabal/pull/3

**Comment by ulysses4ever** [comment]
Actually, disregard that, I'll just kill the line about number of appovers in the Margify config.

**Comment by ulysses4ever** [comment]
Okay, I think I got it: I got the inequality in "updated-at>=2 days ago" the other way round (>= means after that moment in time, and we want before). Waiting for the new test PR to be merged in 30 minutes after merge-me label applied https://github.com/ulysses4ever/cabal/pull/7 after I will say I'm confident this should work.

**Comment by ulysses4ever** [comment]
Yes, worked fine (only I used 30 mins instead of 2 days chillout)

**Comment by ulysses4ever** [comment]
@Mikolaj CHANGELOG updated

**Comment by ulysses4ever** [comment]
Arghhh, not changelog, but contributing.md

File: CONTRIBUTING.md
Comment (ulysses4ever) [reply CONTRIBUTING.md]: No, the commands like refresh/rebase/rebase are a different thing, they work as before.

Comment (Mikolaj) [reply CONTRIBUTING.md]: I'm not being clear. I mean, after "mergifyIO rebase", does it wait 2 more days? That's not fatal, just wondering.

Comment (ulysses4ever) [reply CONTRIBUTING.md]: I think I got you right?.. If you comment on a PR thread `@mergify rebase`, it won't wait before rebasing, because it's a "command" (in Mergify vocabulary), which is not an "action" that Mergify performs on PRs. So, after a comment `@mergify rebase`, Mergify will rebase as before, without waiting 2 days. But it will wait for 2 days before merging the PR (if all other conditions are satisfied).

**Comment by ulysses4ever** [comment]
I forgot to mention the `squash+merge me` label. Will have to get back to it later today.

File: CONTRIBUTING.md
Comment (Mikolaj) [reply CONTRIBUTING.md]: OK, I was interested in the last part of that. If a user, after waiting 2 days, rebases, either with a manual push or with `mergifyIO rebase`, the user may be surprised that extra 2 days of waiting are needed. However, there is usually no good reason to rebase and some of the reasons involve non-trivial changes to CI, github Settings or the code-base, so extra waiting period makes sense.

Comment (Mikolaj) [reply CONTRIBUTING.md]: In any case, we now mention "rebase", so the user should not be surprised.

**Comment by ulysses4ever** [comment]
Added a sentence about `squash`, check it out...

File: CONTRIBUTING.md
Comment (ulysses4ever) [reply CONTRIBUTING.md]: Fixed.

**Review by jneira — COMMENTED** [review]
many thanks, I did not follow the IRC discussion, could we paste/mention it briefly? i cant think why we want to delay merging even more (they usually takes quite time!)

**Comment by ulysses4ever** [comment]
@jneira some members of the team expressed a concern that PR sometimes get merged too fast, and the chillout time after review was a proposed as a countermeasure.

**Review by emilypi — COMMENTED** [review]
I think this is pretty fair

**Review by emilypi — APPROVED** [review]

**Comment by Mikolaj** [comment]
2 days have passed. Time to merge?

**Comment by ulysses4ever** [comment]
Oh wow
> Error: All install methods for ghc 8.0.2.2 failed
https://github.com/haskell/cabal/runs/7448278695?check_suite_focus=true

**Comment by ulysses4ever** [comment]
Mergify says config-changing PRs have to be merged manually. Merging...

**Pull request merged**
